package.json

```diff
@@ -40,18 +40,18 @@
     "eslint-plugin-no-null": "^1.0.2",
     "ignore-styles": "^5.0.1",
     "lerna": "^6.6.2",
-    "mocha": "^10.2.0",
+    "mocha": "^11.0.1",
     "mocha-jenkins-reporter": "^0.4.8",
     "nyc": "^15.1.0",
-    "prettier": "^2.8.8",
-    "reflect-metadata": "^0.1.13",
+    "prettier": "^3.4.2",
+    "reflect-metadata": "^0.2.2",
     "rimraf": "^5.0.1",
     "sinon": "^15.1.0",
     "ts-node": "^10.9.1",
     "typescript": "^5.0.4"
   },
   "engines": {
-    "node": ">=16.11.0",
-    "yarn": ">=1.7.0 <2.x.x"
+    "node": ">=18",
```

Comment:
I assume this will be documented somewhere once we have a release.

Reply:
Yes it will be part of all changelogs.
Theia has dropped support for node 16 so we should do that do.
However, I don't think that this should affect many adopters since 16 is in its EOL phase since September 2023.